.github/workflows/build.yml

```diff
@@ -0,0 +1,54 @@
+name: build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: ${{ matrix.os }} / GHC ${{ matrix.ghc }}
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os:
+          - ubuntu-16.04 # https://launchpad.net/~hvr/+archive/ubuntu/ghc?field.series_filter=xenial
+        ghc:
+          - 7.0.4
+          - 7.2.2
+          - 7.4.2
+          - 7.6.3
+          - 7.8.4
+          - 7.10.3
+          - 8.0.2
+          - 8.2.2
+          - 8.4.4
+          - 8.6.5
+          - 8.8.4
+          - 8.10.3
```

Comment:
This uses the latest version of GHC 8.10, which is fine, but it does mean that it's out of sync with what is advertised [here](https://github.com/haskell-compat/base-compat/blob/035ae45db3f964d50193c5663c068fae265163ec/base-compat/base-compat.cabal#L57). We should keep this up to date.

Reply:
As far as I know, only `haskell-ci` is looking at `tested-with`.  So unless you are using `haskell-ci`, I think there is no important reason to maintain this information in the `.cabal` file.

Reply:
One more thing, if we only want to test the latest minor versions, we can simplify this to:

```yaml
ghc:
  - 7.0.4
  - 7.2.2
  - 7.4.2
  - 7.6.3
  - 7.8.4
  - '7.10'
  - '8.0'
  - '8.2'
  - '8.4'
  - '8.6'
  - '8.8'
  - '8.10'
```
(note that `pre-7.10` still requires the minor version explicitly; also not that the quotes are necessary, so that it's not parsed as a number)

Reply:
I'd be fine with removing the `tested-with` information. Let's just do that.